WalletWasabi.Fluent/ViewModels/AddWallet/AddedWalletPageViewModel.cs

```diff
@@ -42,7 +42,12 @@ private async Task OnNextAsync(WalletCreationOptions options)
 
 		await AutoLoginAsync(options);
```

Comment:
It seems like something is happening on the UI thread, so the animation is laggy